props/prop_base.h

```diff
@@ -1376,6 +1376,32 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
       }
       return true;
     }
+    if (!strcmp(cmd, "swing")) {
+      SaberBase::DoEffect(EFFECT_ACCENT_SWING, 0);
+      Event(BUTTON_NONE, EVENT_SWING);
+      // Since idle sound re-uses hum_player,this will prevent
+      // a swing command from triggering monophonic hum while SaberBase is OFF.
+      if (!SaberBase::IsOn() && !GetWavPlayerPlaying(&SFX_idle)) {
```

Comment:
This seems like it should be fixed in hybrid_font or something (and be guarded by an #ifdef)
Basically, instead of transitioning to the hum sound we should just transition to the idle if the saber is off.
(Assuming we care about supporting idle in monophonic fonts that is...)

Reply:
I'm all for not caring, as i personally haven't come across a monophonic font for a good few years now.
That said, it's nice to be robust.
In the off chance someone uses idle.wav in a common folder (hi!) then choosing a mono font with it is likely.


> This seems like it should be fixed in hybrid_font or something (and be guarded by an #ifdef)

I'll look into that. 
#ifdef ... what's the condition? You mean for SPINS again? I try not to forget.

Reply:
Moved to a function in hybrid_font called CheckStopIdle().